src/platforms/android/configuration/gradle.mdx

```diff
@@ -57,7 +57,7 @@ We expose the following configuration values directly in your `app/build.gradle`
 
 <SignInNote />
 
-```groovy
+````groovy
```

Comment:
I guess my vscode formatter did that for whatever reason 🤷

Reply:
or maybe prettier pre-commit hook? not sure

Reply:
lol wat https://github.com/getsentry/sentry-docs/pull/8613/commits/72a8ffde484ad17e644c8babf312be0f3b60fa47

Reply:
https://github.com/getsentry/sentry-docs/pull/8613/commits/d947dbee764e8887a9a99ea7c982ae70c831b2e7 it keeps doing it :/

Reply:
ugh that's SO weird. I will take a look or ask Evan (who set it up) if I can't figure it out.

Reply:
I see - it's because there are backticks in the code blocks, which are working properly... since everything is working right and this is what the linter wants, I'm going to assume it's ok!